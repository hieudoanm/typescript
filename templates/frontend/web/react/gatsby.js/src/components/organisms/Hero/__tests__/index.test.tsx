import { render } from '@testing-library/react';
import { Hero } from '../index';
import React from 'react';

describe('Hero', () => {
  it('renders correctly', () => {
    const button = render(<Hero title={''} description={''} />);
    expect(button.container).toMatchSnapshot();
  });
});
