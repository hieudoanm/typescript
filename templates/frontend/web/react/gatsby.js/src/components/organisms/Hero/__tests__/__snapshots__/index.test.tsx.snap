// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders correctly 1`] = `
<div>
  <section
    class="py-8 md:py-16"
    id="home"
  >
    <div
      class="container mx-auto"
    >
      <div
        class="mb-8 md:mb-16"
      >
        <div
          class="text-center mx-auto max-w-lg"
        >
          <h1
            class="text-2xl sm:text-4xl font-b162 mb-4 font-bold"
          />
          <h2
            class="text-sm sm:text-lg"
          />
        </div>
      </div>
      <div
        class="relative"
      >
        <div
          class="top-0 flex w-full justify-center pb-16 sm:absolute"
        >
          <a
            class="cursor-pointer"
          >
            <button
              class="rounded-md bg-emerald-500 px-8 py-4 text-white transition duration-500 hover:bg-gray-900 "
              type="button"
            >
              Get Started
            </button>
          </a>
        </div>
        <img
          alt="Banner"
          class="hidden sm:block"
          src="src"
        />
      </div>
    </div>
    <div
      class="relative hidden sm:block"
    >
      <div
        class="absolute bottom-0 -z-10 h-32 w-full bg-[#FFF5ED]/50"
      />
    </div>
  </section>
</div>
`;
